scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -150,6 +158,14 @@ task ExtractTask {
                 --project-id ~{read_project_id} \
                 ~{true='--emit-pls' false='' emit_pls} \
                 ${FILTERING_ARGS}
+
+        # Drop trailing slash if one exists
+        OUTPUT_GCS_DIR=$(echo ~{output_gcs_dir} | sed 's/\/$//')
+
+        if [ -n "${OUTPUT_GCS_DIR}" ]; then
```

Comment:
Since these files are also task outputs, if someone runs with `output_gcs_dir` they'll have two copies of the data, one in the cromwell delocalized results of this task and another in the output bucket?  Do you have ideas about how to handle that?

Reply:
By the cromwell delocalized results, do you mean the files in the "execution directory" (as Terra calls it)? In my mind, I had always thought of those files as "temporary" as in they're usually created within a TTL folder. I'm realizing now that this may be an incorrect assumption that I carried over from my time working with the long reads methods team.

I feel like that might be OK though? Users of the WDL that do not want the duplication can drop the `output_gcs_dir` argument. Users that want the `output_gcs_dir` argument likely only care about the VCF output files so they can clean up the execution directory as needed.